### PR TITLE
Add a skill to better understand bullet lists of items

### PR DESCRIPTION
**Pull request by ae2015**
This is a re-submission of [**PR 652**](https://github.com/instructlab/taxonomy/pull/652).

**Describe the contribution to the taxonomy**

Understand better when a bullet list of items means that the items are needed jointly rather than separately.

- Documents often contain bullet lists of items that one must provide, have available, complete, etc.
- Sometimes Merlinite 7b gets confused whether the listed items must be provided jointly or separately
- Merlinite 7b tends to think that the items are needed separately, even though they are needed jointly
- Here we add a few of such examples, with the correct answers

**Input given at the prompt**

In one example, Merlinite 7b was given the following (self-authored) context document:
```
To apply for public benefit XYZ, provide the following:

* Your full name and date of birth

* Your vehicle license plate number
```
Then the following question was asked:
```
Which of the following is true:

(A) To apply for public benefit XYZ, you must provide both your full name and vehicle license plate number

(B) To apply for public benefit XYZ, you must provide either your full name or vehicle license plate number
```
Most humans would choose (A), but Merlinite 7b chose (B), responding as follows:

**Response from the original model**

```
The correct answer is (B) To apply for public benefit XYZ, you must provide either your full name or vehicle license plate number.
The statement is true because the application process requires one of these pieces of information, but not necessarily both. 
```

**Response from the fine-tuned model**

(did not have an opportunity to fine tune the model)

**Contribution checklist**

- [x] The contribution was tested with `ilab generate`
- [x] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file.

### CODE REVIEW

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
@instructlab-bot generate

**Comment by nerdalert** [comment]
@instructlab-bot generate

**Comment by mingxzhao** [comment]
Marking as approved, SDG and precheck are tested [here](https://github.com/instructlab/taxonomy/pull/756)

File: compositional_skills/linguistics/bullet_lists/attribution.txt
Comment (jjasghar) [inline compositional_skills/linguistics/bullet_lists/attribution.txt:1]: We can merge this as soon as the `attribution.txt` is the correct format. Here is an example: https://github.com/instructlab/taxonomy/pull/1153/files#diff-72966a485aa3a4b2e29a3d310c4024008f96c9c119ec43e878b3ce5668512386

**Review by jjasghar — CHANGES_REQUESTED** [review]
One last change before we can merge this in.

**Comment by alinaryan** [comment]
Thank you for your submission. Here are your results: 
Your PR improved the model performance by 0.05000000000000071 points. 
The model's new score is: 8.98 
 The old score was: 8.93 
|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||| 89.8%

**Comment by alinaryan** [comment]
Thank you for your submission. Here are your results: 
Your PR improved the model performance by 0.23999999999999844 points. 
The model's new score is: 9.2 
 The old score was: 8.96 
|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||| 92.0%

File: compositional_skills/linguistics/bullet_lists/attribution.txt
Comment (ae2015) [reply compositional_skills/linguistics/bullet_lists/attribution.txt:1]: @jjasghar I updated `attribution.txt` as needed, thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issue(s).

**Review by jjasghar — APPROVED** [review]

**Pull request merged**